types/ckeditor__ckeditor5-ui/index.d.ts

```diff
@@ -0,0 +1,59 @@
+// Type definitions for @ckeditor/ckeditor5-ui 27.0
+// Project: https://github.com/ckeditor/ckeditor5/tree/master/packages/ckeditor5-ui
+// Definitions by: Federico Panico <https://github.com/fedemp>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.2
```

Comment:
I don't remember if we have discussed this high requirements previously.
Is this really a required version? (this is top notch version)

Reply:
Yeah, this is worth re-thinking unless you're using a 4.2 only feature 👍🏻

Reply:
This is because this package depends on `@ckeditor/ckeditor5-utils` and `utils` depends on 4.2 because of `ResizeObserverEntry`.

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52322#discussion_r611264797